src/Development/IDE/Core/Rules.hs

```diff
@@ -29,6 +29,7 @@ module Development.IDE.Core.Rules(
 import Fingerprint
 
 import Data.Binary
+import Util
```

Comment:
Where does this come from?

Reply:
The `ghc` package.

Reply:
The GHC package exporting Util was never a good idea... Can we fix it upstream...